orchestration/OrchestrationService/CustomScriptExecution.ps1

```diff
@@ -0,0 +1,257 @@
+Class CustomScriptExecution {
+
+    hidden $scriptType = "";
+    hidden $command = "";
+    hidden $arguments = @{};
+    
+    # Used to store results from the script execution
+    $Result = "";
```

Comment:
Try to avoid the use of global variables that hold processed data, if your function is processing something and such something needs to be return, use an explicit return instead.

Reply:
Removed all global variables.